rest-fights/src/main/java/io/quarkus/sample/superheroes/fight/client/WebAuthnRegisterResponse.java

```diff
@@ -0,0 +1,21 @@
+package io.quarkus.sample.superheroes.fight.client;
+
+import jakarta.ws.rs.BeanParam;
+import jakarta.ws.rs.FormParam;
+
+import io.vertx.core.json.JsonObject;
+
+//Copied from quarkus code
+public class WebAuthnRegisterResponse extends WebAuthnResponse {
+  /**
+   * Corresponds to the JSON {@code response.attestationObject} field
+   */
+  @FormParam("webAuthnResponseAttestationObject")
+  public String webAuthnResponseAttestationObject;
+
+  @Override
+  protected void toJsonObject(JsonObject response) {
```

Comment:
Why using generic `JsonObject` rather than creating an object that represents the proper structure?

Reply:
https://github.com/quarkusio/quarkus/blob/main/extensions/security-webauthn/runtime/src/main/java/io/quarkus/security/webauthn/WebAuthnRegisterResponse.java Used the same one available here. Since `WebAuthnRegisterResponse` is only available if you have the `WebAuthn` extension and I didn't want to add the extension to rest-fights, I copied it from there